torch/_dynamo/output_graph.py

```diff
@@ -2308,6 +2301,12 @@ def specialized_dispatch(*args: Any, **kwargs: Any) -> Any:
                 # This is safe because we pre-process name to be unique
                 self.install_global_unsafe(name, compiled_fn)
 
+            # Cleanup old_fake_mode to release weakrefs
+            # Export uses this fake mode again, so don't clear if export
+            # TODO: If shouldn't be needed when torch._dynamo.export is cleaned up.
```

Comment:
For this comment, can we link to an issue or assign (@ someone here) so that we have more viz to clean this?